x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/jobs_list/jobs_list.js

```diff
@@ -126,7 +138,10 @@ class JobsListUI extends Component {
       ),
       onSelectionChange: this.props.selectJobChange
     };
-
+    // Adding 'width' props to columns for use in the Kibana management jobs list table
+    // The version of the table used in ML > Job Managment depends on many EUI class overrides that set the width explicitly.
+    // The ML > Job Managment table won't change as the overwritten class styles take precedence, though these values may need to
+    // be updated if we move to always using props for width.
```

Comment:
css overrides where originally used because it wasn't possible to add widths to the column objects.
it would be good to remove the css overrides and have all widths here.
This could be done in a follow up PR